test/specs/notification_spec.rb

```diff
@@ -2,9 +2,18 @@
 require 'pry'
 
 class Jackal::Slack::Notification
-  def post_to_slack(description, attachments)
-    $attachments = attachments
-    $description = description
+  alias_method :og_execute, :execute
```

Comment:
wait, is this :complete supposed to be on the result payload (from the MessageStore)?
Because I don't see that key on the result from a valid payload...
Here for example: https://github.com/carnivore-rb/jackal-slack/pull/5/files#diff-b2e86fb46293f60ab8450f2230754554R29